tf/do/files/rclone.sh

```diff
@@ -26,14 +26,14 @@ for s in 3 2 1; do
       shard=${s}
       # download shard db
       echo rclone syncing harmony_db_${shard}
-      rclone sync -P mainnet:pub.harmony.one/mainnet/harmony_db_${shard} /root/do-user/harmony_db_${shard}
+      rclone sync -P mainnet:pub.harmony.one/mainnet.min/harmony_db_${shard} /root/harmony_db_${shard}
```

Comment:
look at my change in aws.1/files/rclone.sh, and aws.1/go.sh, I make it a parameter for rclone.sh, so that it can support both.  But anyway, you can change it later.

Reply:
Got you, Thx. Will get it improved tmr.

Reply:
read through the changes, it is a better design indeed. Implementing them in my code.